Server/Server.Core/Server.Core.Static/ArgParser.hpp

```diff
@@ -4,52 +4,48 @@
 #include <FileLogger.hpp>
 
 /**
- * @class ArgParser.
- * @brief This class parses arguments with which application is started.
- * @details
- * For example: "--serverport 5432 --port 65001 --dbname juniorgram --hostaddr 127.0.0.1 --user postgres --password postgres".
- * Sending the port to the Server class constructor and list of all arguments to the PostgreAdapter Instance Method.
- */
+* @class ArgParser.
+* @brief This class parses arguments with which application is started.
+* @details
+* For example: "--serverport=65001 --port=5432 --dbname=juniorgram --hostaddr=127.0.0.1 --user=postgres --password=postgres".
+* Sending the port to the Server class constructor and list of all arguments to the PostgreAdapter Instance Method.
+*/
 class ArgParser
 {
 public:
     /**
-     * @brief It's deleted default constructor.
-     * @details There is no need to use this.
-     */
+    * @brief It's deleted default constructor.
+    * @details There is no need to use this.
+    */
     ArgParser() = delete;
    
     /**
-     * @brief It's deleted copy constructor.
-     * @details There is no need to use this.
-     */
+    * @brief It's deleted copy constructor.
+    * @details There is no need to use this.
+    */
     ArgParser(const ArgParser&) = delete;
    
     /**
-     * @brief Used to take parameters from main function.
-     * @details Parses arguments which were set when application was run.
-     * @param int argc(amount of arguments), const char** argv(C-string arguments).
-     */
+    * @brief Used to take parameters from main function.
+    * @details Parses arguments which were set when application was run.
+    * @param int argc(amount of arguments), const char** argv(C-string arguments).
+    */
     explicit ArgParser(int argc, const char** argv) 
     {
         _argParser = argparse::ArgumentParser("ServerArguments", "0.6",
         argparse::default_arguments::none);
 
+        _argParser.set_assign_chars("=");
         _argParser.add_description("Server side of juniorgram messenger. Just have a fun!");
         _argParser.add_epilog("Some parts of the program may be configured by default.");
 
         _argParser.add_argument("--serverport")
             .default_value<std::string>("65001")
             .help("Set the port value for server needs")
             .action([](const std::string& value) -> std::string {
-                uint16_t port = static_cast<uint16_t>(std::stoi(value));
+                auto port = static_cast<uint32_t>(std::stoi(value));
```

Comment:
Based on previous implementation, if the port exceeded a limit of type uint16_t, the default port is returned.
But because we have a limit of type 16_t, provincializing the port with a value exceeding type 16_t will cause an overflow problem and the port will be initialized with a random value of type 16_t.
The question arises, why do we need a limit check if it does not work?
Consequently, this check now.